welearn_datastack/modules/qdrant_handler.py

```diff
@@ -34,31 +35,23 @@ def classify_documents_per_collection(
     """
     tmp_collections_names_in_qdrant = qdrant_connector.get_collections().collections
     collections_names_in_qdrant = [c.name for c in tmp_collections_names_in_qdrant]
+    model_name_collection_name = {
+        x.split("_")[3]: x for x in collections_names_in_qdrant
```

Comment:
:baguette_bread: just because we know that the model is always at the end : 
```suggestion
        x.split("_")[-1]: x for x in collections_names_in_qdrant
```